apps/web/src/payload/fields/Inputs/TextArea/TextArea.config.ts

```diff
@@ -0,0 +1,50 @@
+import type { Block } from 'payload/types';
+
+const TextAreaBlock: Block = {
```

Comment:
TextArea doesn't exist as a refract hook field, so I had to create my own for the `FormBlock`. It might be good to create a dedicated TextArea input in the refract repo